.github/workflows/scripts/prepare-test-summary.js

```diff
@@ -8,6 +8,7 @@ const {
 	E2E_PPTR_SUMMARY_PATH,
 	SHA,
 	PR_NUMBER,
+	E2E_GRAND_TOTAL,
 } = process.env;
 
```

Comment:
This file has become a big one (+250 code lines), do you think we can split it somehow @rodelgc? 

Perhaps we could have:
- One file with the functions to retrieve stats (`getAllureSummaryStats`, `getPuppeteerStats`...)
- Another file to prepare the static attributes of the report (for example `createAPITableRow`, `createE2ETableRow`, `addSummaryFooter`...)
- Another file to handle warnings/errors

I don't have enough knowledge of this code so I'm just thinking out loud, but would something like this make sense? Of course, we could follow another criteria to split the code if it's more appropriate.